src/main/java/clients/symphony/api/PresenceClient.java

```diff
@@ -5,29 +5,45 @@
 import exceptions.SymClientException;
 import exceptions.UnauthorizedException;
 import java.util.List;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
 import javax.ws.rs.client.Entity;
 import javax.ws.rs.client.Invocation;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.GenericType;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
-import model.Presence;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
 import model.UserPresence;
+import model.UserPresenceCategory;
 
+@RequiredArgsConstructor
```

Comment:
what does this class mean?

Reply:
It will generate a constructor including `private final` fields only